cli/src/main/java/ai/wanaku/cli/main/support/WanakuPrinter.java

```diff
@@ -311,6 +317,103 @@ public void printSuccessMessage(String message) {
         }
     }
 
+    /**
+     * Prints Markdown text with terminal styling and formatting.
+     *
+     * <p>Parses and renders Markdown text with ANSI color codes and text formatting
```

Comment:
Just a heads up that these `<p>` and `</p>` tend to cause the Javadoc generation to fail. I'd recommend doing a dist build to make sure none of these cause problems.